tests/test_flow_signal.py

```diff
@@ -12,8 +17,15 @@ def test_signal_usecase(self):
         process = SignalFlow.process_cls.objects.get()
         task_test_signal.send(sender=self, process=process)
 
+        user = User.objects.create(username="test")
+
+        with FlowTest(SignalFlow) as flow:
```

Comment:
The primary goal of the test is not to have to debug the test code itself.

viewflow test framework can't be used inside other viewflow tests.